src/smt/set_defaults.cpp

```diff
@@ -178,6 +178,15 @@ void SetDefaults::setDefaultsPre(Options& opts)
                      options::UnsatCoresMode::SAT_PROOF,
                      "enabling proofs");
     }
+    if (opts.proof.proofFormatMode == options::ProofFormatMode::ALETHE
+        && opts.proof.proofGranularityMode
```

Comment:
It's slightly dangerous to compare here, can you at least comment that this assumes that granularity modes are stored in order of precision?